Bottom: 5069e6b8649c3e6d525165294f10e197978fc688
Top:    20520676136d20af4114f8d6b4bec8449d4e3b0a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:02:15 +0100

Refresh of add-raw-polygons-support-for-t

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 9460091..4e1c29a 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -670,6 +670,7 @@ ghid_fill_polygon (hidGC gc, int n_coords, int *x, int *y)
   gdk_draw_polygon (gport->drawable, priv->u_gc, 1, points, n_coords);
 }
 
+void
 ghid_fill_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 {
   common_fill_pcb_polygon (gc, poly, clip_box);
